### ISSUE TITLE
The text on the application interface is difficult to read

### ISSUE THREAD

**Issue by FeiLZX3Km** [aleksey-hoffman/sigma-file-manager#335]
### Problem description

The text on the application interface is difficult to read. Could you please increase the contrast between the text and the background, and also adjust the brightness to make it clearer?

### Steps to reproduce

1. 

### Operating system

windows 11

### App version

v2

**Comment by Dark-Ai-py** [comment]
> Same on Linux Mint, contrast is very low for both light and dark mode but the effect is worse for the dark theme. You really need to use white/slightly dimmed white instead of low-key colors and tones.
> 
>     * Or you introduce a `Dark-gray` theme variant to replace the dark theme. Example:
>       https://raw.githubusercontent.com/Draekko-RAND/gimp-themes/gimp-2_10/screenshots/sog.png
> 
> 
> > The gray theme displayed in the link above could work nicely for _day and night_. If you disagree, this gray could be a nice replacement for the bland "light theme", imo.
> 
>     * Finally, would be great to customize Sigma's appearance with a `style.css` file, if possible.

Yeah being able to change the theme colors and such would be so much easier
